eth-contracts/contracts/service/ServiceProviderFactory.sol

```diff
@@ -78,13 +116,9 @@ contract ServiceProviderFactory is RegistryContract {
             _delegateOwnerWallet
         );
 
-        uint currentlyStakedForOwner = stakingContract.totalStakedFor(owner);
-        uint minStakeAmount = stakingContract.getMinStakeAmount();
-
-        require(
-            currentlyStakedForOwner >= minStakeAmount,
-            "Minimum stake amount not met");
+        validateAccountStakeBalances(owner);
```

Comment:
just to double check, the require in the `validateAccountStakeBalances` will revert this entire tx if the new bounds are invalid, either too much or too little staked for the # of services. which means currently if you have a set of services, it must meet the criteria and be in the happy case

Reply:
that is correct, it will not be allowed to complete